client/components/TheGame.jsx

```diff
@@ -15,8 +14,7 @@ class TheGame extends Component {
       player1: '',
       player2: '',
       style1: {},
-      style2: {},
-      gameArr: gameArr
+      style2: {}
```

Comment:
probably unimportant but it might be good to get some context on what style is. is that for the colour of each user?